### PR TITLE
Minor updates, mostly around MZSK current status.

### PR DESCRIPTION
**Pull request by shane-kerr**

### CODE REVIEW

File: Yeti-experience/draft-song-yeti-testbed-experience-01.txt [hunk 1]
```diff
@@ -669,29 +669,32 @@ Internet-DraftExperiences from Root Testbed in the Yeti DNSDecember 2015
...
+   some problem with older versions of the Unbound resolver).
 
-   According to the Yeti experiment protocol, a lab test was done to
-   verify the concept and validity of Multi-ZSK.  The purpose of the
-   test is two-fold: 1) To test whether this proposal can be implemented
-   by current DNS protocol & software (to see if there should be some
-   extra modification to protocol or software), and 2) To demonstrate
-   the impact of Multi-ZSK proposal to the current root system.
+4.2.1.  MZSK lab experiment
+
```
Comment (songlinjian) [inline Yeti-experience/draft-song-yeti-testbed-experience-01.txt]: I think after we done MZSK experiment on Yeti testbed, we do not need to elaborate the lab experiment. What do you think ?

File: Yeti-experience/draft-song-yeti-testbed-experience-01.txt [hunk 2]
```diff
@@ -755,6 +757,38 @@ Internet-DraftExperiences from Root Testbed in the Yeti DNSDecember 2015
...
+Song & Kerr            Expires September 22, 2016              [Page 14]
+

+Internet-DraExperiences from Root Testbed in the Yeti DNS Pr  March 2016
+
+
+   authority server, and issues with IXFR.  These will be detailed in a
+   seperate report.
+
```
Comment (songlinjian) [inline Yeti-experience/draft-song-yeti-testbed-experience-01.txt]: I think we should put the findings in section 5

Comment (shane-kerr) [inline Yeti-experience/draft-song-yeti-testbed-experience-01.txt]: Davey,

At 2016-03-21 18:15:34 -0700
"Linjian Song (Davey)" <notifications@github.com> wrote:

> > +
> > +   The MZSK Yeti part of the experiment is still in progress, and is
> > +   expected to finish in 2016-04.  There have been some findings,
> > +   including discovering that the large responses caused problems on on
> > +
> > +
> > +
> > +
> > +Song & Kerr            Expires September 22, 2016              [Page 14]
> > +
> > +Internet-DraExperiences from Root Testbed in the Yeti DNS Pr  March 2016
> > +
> > +
> > +   authority server, and issues with IXFR.  These will be detailed in a
> > +   seperate report.
> > +  
> 
> I think we should put the findings in section 5

Then we have to wait for the findings before publishing this draft as
an RFC. I thought we were in a hurry?

--
Shane

**Comment by songlinjian** [comment]
> In order to let people know the technical activities in Yeti DNS project, this document reports and discusses issues on root DNS services, based on experiences so far from the experiments in the Yeti DNS project.

If we decide to document the experiment and technical findings separately in others IETF drafts or document, then we should think seriously whether we should change the motivation and title of this document. 

For example:  "This document describe motivation , problem statement, the technical structure of Yeti testbed as well as the technical finds and considerations when building such research alternative root. The further discussion of yeti experiments will be covered by separate  document or individual draft."

**Pull request merged**